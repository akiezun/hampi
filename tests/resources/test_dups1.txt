S = X;
A = B;
X = "x"; 
B = "B"; 
