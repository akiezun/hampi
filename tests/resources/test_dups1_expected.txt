A = B  ;
B = "B"  ;
S = X  ;
X = "x"  ;
